### ISSUE TITLE
FileStorageService creates folders + a file instead of a single file when file name has forward slashes

### ISSUE THREAD

**Issue by hsilva-keep** [keeps/roda#300]
Example file name: 
```
org.roda.core.plugins.plugins.antivirus.AntivirusPlugin@ClamAV 0.98.7/21498/Tue Apr 19 03:35:25 2016.agent.premis.xml
```

**Comment by hsilva-keep** [comment]
Solved since 9e0eb8381e99ae7589a27dcf1a33f8deed321cea

**Comment by hsilva-keep** [comment]
Solved since 9e0eb8381e99ae7589a27dcf1a33f8deed321cea

**Comment by hsilva-keep** [comment]
Solved since 9e0eb8381e99ae7589a27dcf1a33f8deed321cea
